### PR TITLE
Update resourcemanager-streamanalytics-readme.md

### PR DESCRIPTION
**Pull request by JasonWHowell**
Fix metadata

### CODE REVIEW

**Comment by JasonWHowell** [comment]
CC: @spelluru

**Comment by spelluru** [comment]
Thanks @JasonWHowell 

Looks like we will have to update the ms.author attribute. It should be set to: jfree. Could you please update it too? Thanks! 

https://repos.opensource.microsoft.com/people?q=joshfree

**Comment by JasonWHowell** [comment]
#sign-off

**Comment by JasonWHowell** [comment]
@spelluru can you please review the PR with the review button to approve.

**Review by spelluru — APPROVED** [review]
Thanks @JasonWHowell

**Comment by JasonWHowell** [comment]
@spelluru thanks. I see now "by reviewers with write access." which we don't have.

**Comment by JasonWHowell** [comment]
@Taojunshen can you merge this one? Its just a metadata change.

**Comment by JasonWHowell** [comment]
@Taojunshen  could you merge this one?

**Comment by JasonWHowell** [comment]
@danieljurek  can you merge this one? I'm not sure who can help merge in this repo.

File: docs-ref-services/preview/resourcemanager-streamanalytics-readme.md [hunk 1]
```diff
@@ -107,7 +106,7 @@ For details on contributing to this repository, see the [contributing guide](htt
 <!-- LINKS -->
 [survey]: https://microsoft.qualtrics.com/jfe/form/SV_ehN0lIk2FKEBkwd?Q_CHL=DOCS
 [docs]: https://azure.github.io/azure-sdk-for-java/
-[jdk]: https://docs.microsoft.com/java/azure/jdk/
+[jdk]: /java/azure/jdk/
```
Comment (srnagar) [inline docs-ref-services/preview/resourcemanager-streamanalytics-readme.md:109]: Should this be the absolute link? https://learn.microsoft.com/en-us/azure/developer/java/fundamentals/java-support-on-azure

**Review by danieljurek — APPROVED** [review]

**Comment by danieljurek** [comment]
Merging. Changes that aren't in the package's README.md file in the azure-sdk-for-java repo will be overwritten on the next publish. To ensure your changes persist, update that file too.

**Pull request merged**